### DISCUSSION TITLE
I use two different yml files and GitHub actions to render books in two languages

### DISCUSSION THREAD

**Discussion by sbalci** [quarto-dev/quarto-cli#739] [Show and Tell]
Hi, 

Just saw this category to show things.

I am using quarto to make an educational book. And I use two different yml files and GitHub actions to render books in two languages.
As discussed here: https://github.com/quarto-dev/quarto-cli/issues/275#issuecomment-1073259436 
project GitHub action: https://github.com/patolojiatlasi/patolojiatlasi.github.io/blob/main/.github/workflows/Quarto-Render-Bilingual-Book-Push-Tweet-Updates.yml 

Best wishes

Serdar

**Comment by cderv** [comment]
Thanks for sharing ! I have added your website and workflow among community example in https://github.com/quarto-dev/quarto-actions/tree/main/examples#repositories-using-quarto-actions

Hope that is ok with you ! (I'll revert otherwise)

**Reply by sbalci** [reply]
Sure :)
I am glad you found it worth sharing :)
